packages/fiori/src/WizardPopover.hbs

```diff
@@ -29,4 +32,7 @@
 			@click="{{_closeRespPopover}}"
 		>Cancel</ui5-button>
 	</div>
+
+	<span id="{{_id}}-actions" class="ui5-hidden-text">{{actionSheetActionsText}}</span>
+	<span id="{{_id}}-steps" class="ui5-hidden-text">{{actionSheetStepsText}}</span>
```

Comment:
no need for this, as mentioned we have support for aria-label on popup level so no need of invisible span